src/Agent.Sdk/Knob/AgentKnobs.cs

```diff
@@ -272,5 +272,12 @@ public class AgentKnobs
             new RuntimeKnobSource("DISABLE_OVERRIDE_TFVC_BUILD_DIRECTORY"),
             new EnvironmentKnobSource("DISABLE_OVERRIDE_TFVC_BUILD_DIRECTORY"),
             new BuiltInDefaultKnobSource("false"));
+
+        public static readonly Knob DisableNode6DeprecationWarning = new Knob(
```

Comment:
Could it make sense to add a comment about removing this Knob once we will drop support of the Node 6 handler?

Reply:
Fixed, thanks!